### ISSUE TITLE
Unable to overwrite HTML lang attribute

### ISSUE THREAD

**Issue by torgeirl** [devilry/trix2#114]
It seems both `2.1.4` and `3.0.0b4` default to `lang="en"` despite `trix/project/production/settings.py` containing `LANGUAGE_CODE = 'nb'`, and it isn't possible to overwrite by adding `LANGUAGE_CODE = 'nb'` to our local `trix_settings` file.

**Comment by Levijatan** [comment]
Searched up where it was hardcoded and quickly fixed it.
